src/lighting/lighting-controller.js

```diff
@@ -400,6 +407,29 @@ class LightingController {
   }
 
   /**
+   * Uses the Hue API to change the color of all lights within range.
+   */
+  colorLights(xColor) {
+    var _self = this;
+
+    switch(xColor){
```

Comment:
Per having the ".create().on().hue(rgbValue).sat(255)" commands in the switch statement, I did this because I wasn't sure if we would have more complex color/theme calls down the line (e.g. "change the theme to ocean") which could affect multiple lights, but we can always cross that bridge when we come to it.  I went ahead and simplified the switch statement and pulled out the ".create().on().hue(rgbValue).sat(255)" commands below.

Per incorporating the onecolor NPM, that's a great idea...the only reason I would shy away from that is because I've found the RGB representations of colors to be a little off.  Hue values (0 to 65535) seem to be more close representations of color in the Philips Hue lighting system.  That's why I only used it once in my code, for white, and more for reference only.